include/dlpack/dlpack.h

```diff
@@ -67,7 +67,7 @@ typedef struct {
  */
 typedef struct {
   /*! \brief Type code of base types */
-  uint8_t code;
+  DLDataTypeCode code;
```

Comment:
The reason to keep it as uint8_t is because enum is going to become int, which makes it 8 bytes, while original field packs into 4 bytes